index.js

```diff
@@ -33,6 +34,7 @@ const startAction = async (inputJson) => {
 
   const priorityMapper = new Map(Object.entries(utils.populateMap(PRIORITY_MAPPER)));
   const reportPairsMapper = utils.populateMap(REPORT_INPUT_KEYS);
+  const labels = ISSUE_LABELS_MAPPER.length !== 0 ? { labels: ISSUE_LABELS_MAPPER.split(',') } : { labels: '' };
```

Comment:
Does this pass the JSON validator https://github.com/camelotls/actions-jira-integration/blob/c7ea74e86a97ad32c03a3982ad343e3b9ee2ac37/utils/helper.js#L25-L27 since we expect an array? 

In case of `ISSUE_LABELS_MAPPER.length == 0`, shouldn't we have `{ labels: [] }` (or something like this - haven't tested it)?

Reply:
Updated based on the above comment